docs/MANUAL.md

```diff
@@ -0,0 +1,293 @@
+# Kodelet User Manual
+
+Kodelet is a lightweight agentic SWE Agent that runs as an interactive CLI tool in your terminal. It is capable of performing software engineering and production operating tasks.
+
+## Installation
+
+### Using Install Script
+
+```bash
+curl -sSL https://raw.githubusercontent.com/jingkaihe/kodelet/main/install.sh | bash
+```
+
+### Prerequisites
+
+For running locally or building from source:
+- Go 1.24 or higher
+
+## Updating
+
+To update Kodelet to the latest version:
+
+```bash
+kodelet update
+```
+
+To install a specific version:
+
+```bash
+kodelet update --version 0.0.24.alpha
+```
+
+## Usage Modes
+
+Kodelet supports several usage modes depending on your needs.
+
+### One-shot Mode
+
+Perfect for quick queries and automation:
+
+```bash
+# Basic one-shot query
+kodelet run "your query"
+
+# One-shot query with conversation persistence
+kodelet run "your query"                     # saved automatically
+kodelet run --resume CONVERSATION_ID "more"  # continue a conversation
+kodelet run --no-save "temporary query"      # don't save the conversation
+```
+
+### Interactive Chat Mode
+
+For extended conversations and complex tasks:
+
+```bash
+kodelet chat
+kodelet chat --plain
+```
+
+### Watch Mode
+
+Monitor file changes and automatically process files with special "@kodelet" comments:
+
+```bash
+kodelet watch [--include "*.go"] [--ignore ".git,node_modules"] [--verbosity level] [--debounce ms]
+```
+
+Options:
+- `--ignore` or `-i`: Directories to ignore (default: `.git,node_modules`)
+- `--include` or `-p`: File pattern to include (e.g., `*.go`, `*.{js,ts}`)
+- `--verbosity` or `-v`: Verbosity level (`quiet`, `normal`, `verbose`)
+- `--debounce` or `-d`: Debounce time in milliseconds (default: 500)
+
```

Comment:
document the `kodelet resolve` and `kodelet pr-respond` feature here

run `kodelet resolve --help` and `kodlet pr-respond --help` if you are unfamiliar with the feature

Reply:
also briefly refer the link to the `./docs/MANUAL.md` in the context for better understanding